glm/ext/quaternion_common.inl

```diff
@@ -104,7 +104,7 @@ namespace glm
         {
             // Graphics Gems III, page 96
             T angle = acos(cosTheta);
-            T phi = angle + k * glm::pi<T>();
+            T phi = angle + static_cast<float>(k) * glm::pi<T>();
```

Comment:
Why hardcoding it using single precision floating point? As far as I know, `T` could be `long double`. It should use `static_cast<T>(k)` here at least.

Reply:
This issue is fixed in master branch